packaging/nntrainer.spec

```diff
@@ -215,6 +215,13 @@ Requires:        capi-machine-learning-training-devel = %{version}-%{release}
 %description -n capi-machine-learning-training-devel-static
 Static library of capi-machine-learning-training-devel package.
 
+%package -n capi-machine-learning-training-tizen-internal-devel
```

Comment:
DO NOT ADD "capi" prefix for non public API packages.

Reply:
Updated to nntrainer-tizen-internal-devel :)